demos/jobs/jobs_demo_mosquitto/jobs_demo_mosquitto.c

```diff
@@ -125,21 +125,26 @@ static void usage( const char * programName )
              );
     fprintf( stderr,
              "\nusage: %s "
-             "[-o] -n name -h host [-p port] {--cafile file | --capath dir} --certfile file --keyfile file\n"
+             "[-o] -n name -h host [-p port] {--cafile file | --capath dir} --certfile file --keyfile file [--pollinv seconds] [--updateinv seconds]\n"
              "\n"
              "-o : run once, exit after the first job is finished.\n"
              "-n : thing name\n"
              "-h : mqtt host to connect to.\n"
              "-p : network port to connect to. Defaults to %d.\n",
              programName, DEFAULT_MQTT_PORT );
     fprintf( stderr,
-             "--cafile   : path to a file containing trusted CA certificates to enable encrypted\n"
-             "             certificate based communication.\n"
-             "--capath   : path to a directory containing trusted CA certificates to enable encrypted\n"
-             "             communication.  Defaults to %s.\n"
-             "--certfile : client certificate for authentication in PEM format.\n"
-             "--keyfile  : client private key for authentication in PEM format.\n\n",
+             "--cafile    : path to a file containing trusted CA certificates to enable encrypted\n"
+             "              certificate based communication.\n"
+             "--capath    : path to a directory containing trusted CA certificates to enable encrypted\n"
+             "              communication.  Defaults to %s.\n"
+             "--certfile  : client certificate for authentication in PEM format.\n"
+             "--keyfile   : client private key for authentication in PEM format.\n",
              DEFAULT_CA_DIRECTORY );
+    fprintf( stderr,
+             "--pollinv   : after this many idle seconds, request a job.  Default is not to poll.\n"
+             "--updateinv : after this many seconds running a job, resend the current status to the jobs service.\n"
+             "              Default is not to resend.\n\n"
```

Comment:
Without reading the code, it was not immediately obvious that 0 was mapped to this default.
```suggestion
             "              Default value is 0, which disables periodic resends.\n\n"
```
Feel free to change the wording or document the struct instead. Same also applies to `pollinv`.

Reply:
changed